tt_metal/hw/ckernels/grayskull/metal/llk_api/llk_unpack_tilize_api.h

```diff
@@ -130,7 +130,7 @@ inline void llk_unpack_tilizeA_B_mop_config(const std::uint32_t num_faces) {
     }
```

Comment:
will need condition for `zero_srcA_reduce` like the wormhole version. @rtawfik01 is there anything else we would need to do on grayskull?

Reply:
https://github.com/tenstorrent/tt-metal/pull/12238 Waiting for this merge.